ansible/inventory/group_vars/all/default.yml

```diff
@@ -0,0 +1,17 @@
+---
+hostname: ag
+docker_build: false
+
+quickfeed_user: autograder
+quickfeed_group: autograder
+quickfeed_home: /home/autograder
+
+quickfeed_root: {{ quickfeed_home }}/quickfeed
+quickfeed_ui_root: {{ quickfeed_root }}/public
+quickfeed_bin: {{ quickfeed_home }}/go/bin
```

Comment:
The `quickfeed_bin` is the same as `GOPATH/bin` and `GOPATH/bin` is added to `PATH` by `name: Define PATH facts`. Wouldn't it be enough to use `PATH` in `ExecStart`?

Reply:
True, I'll change that.